apps/meteor/client/views/room/contextualBar/RoomFiles/RoomFilesWithData.tsx

```diff
@@ -14,8 +14,18 @@ const RoomFilesWithData = () => {
 	const room = useRoom();
 	const { closeTab } = useRoomToolbox();
 	const [text, setText] = useState('');
+	const [debouncedText, setDebouncedText] = useState('');
 	const [type, setType] = useLocalStorage('file-list-type', 'all');
 
+	// Debounce the search text to prevent excessive API calls
```

Comment:
Only use comments if the code after it is not easy understandable (that is not the case here)